command/shell_test.go

```diff
@@ -62,5 +65,18 @@ func TestShellExecuteCommand(t *testing.T) {
 
 		})
 
+		Convey("canceling the context should cancel the command", func() {
+			cmd := &shellExec{
+				Script:     "sleep 1",
+				WorkingDir: testutil.GetDirectoryOfFile(),
+			}
+			ctx, cancel := context.WithTimeout(context.Background(), time.Millisecond)
+			defer cancel()
+
+			err := cmd.Execute(ctx, comm, logger, conf)
+			assert.Contains(err.Error(), "shell command interrupted")
+			assert.NotContains(err.Error(), "error while stopping process")
```

Comment:
It's fine to ship this as is, but I'd be consistent with goconvey and use the So() assertions for these rather than testify assertions, but that's a minor quibble.